ddl/column.go

```diff
@@ -1029,8 +1069,10 @@ func (w *updateColumnWorker) fetchRowColVals(txn kv.Transaction, taskRange reorg
 	taskDone := false
 	var lastAccessedHandle kv.Handle
 	oprStartTime := startTime
+	var rowIdx int
 	err := iterateSnapshotRows(w.sessCtx.GetStore(), w.priority, w.table, txn.StartTS(), taskRange.startHandle, taskRange.endHandle, taskRange.endIncluded,
 		func(handle kv.Handle, recordKey kv.Key, rawRow []byte) (bool, error) {
+			rowIdx++
```

Comment:
reorganization work is parallelly done by several updateColumnWorker, each worker for a small splited kv range. I think computing the row number here is not precious. how about just abandoning the row number in the error message?

Reply:
It will be not compatible with mysql's error message. This is not important problem. Sometimes user will want to know which row modify error.

Reply:
For parallel computation, the row count is not the only choice, we can change it to handle's string output

Reply:
Agreed with @AilinKid 

For error message, we don't have to make 100% compatible with MySQL(to be honest, the error message of MySQL is not well-designed). It's all right to use a different message as long as the error code is the same.

Reply:
ok